specification/api-tracing.md

```diff
@@ -386,6 +387,18 @@ Attributes SHOULD preserve the order in which they're set. Setting an attribute
 with the same key as an existing attribute SHOULD overwrite the existing
 attribute's value.
 
+Attribute values expressing a numerical value of zero or an empty string are
+considered meaningful and MUST be stored and passed to the exporter.
+Attribute values of `null` are considered to be not set and discarded as if
```

Comment:
```suggestion
Attribute values of `null` MUST NOT be set and discarded as if
```

Reply:
I would allow attempting to set null values but discard them. This spares users a lot of null checks.

Reply:
fair :+1: I agree